gtars-igd/src/lib.rs

```diff
@@ -8,3 +8,396 @@ pub mod consts {
     pub const IGD_CREATE: &str = "create";
```

Comment:
Is there a reason we are storing CLI consts in the library crate and not the CLI wrapper?